assets/xml/objects/object_boss01.xml

```diff
@@ -1,189 +1,232 @@
+        <Limb Name="gOdolwaSwordLimb" Type="Standard" EnumName="ODOLWA_LIMB_SWORD" Offset="0xEEE0" />
+        <Limb Name="gOdolwaHeadLimb" Type="Standard" EnumName="ODOLWA_LIMB_HEAD" Offset="0xEEEC" />
+        <Limb Name="gOdolwaRightEarringRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_EARRING_ROOT" Offset="0xEEF8" />
+        <Limb Name="gOdolwaRightEarringLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_EARRING" Offset="0xEF04" />
+        <Limb Name="gOdolwaLeftEarringRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_EARRING_ROOT" Offset="0xEF10" />
+        <Limb Name="gOdolwaLeftEarringLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_EARRING" Offset="0xEF1C" />
+        <Limb Name="gOdolwaRightHairRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_HAIR_ROOT" Offset="0xEF28" />
+        <Limb Name="gOdolwaRightHairBaseLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_HAIR_BASE" Offset="0xEF34" />
+        <Limb Name="gOdolwaRightLowerHairRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_LOWER_HAIR_ROOT" Offset="0xEF40" />
+        <Limb Name="gOdolwaRightHairMiddleLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_HAIR_MIDDLE" Offset="0xEF4C" />
+        <Limb Name="gOdolwaRightHairTipLimb" Type="Standard" EnumName="ODOLWA_LIMB_RIGHT_HAIR_TIP" Offset="0xEF58" />
+        <Limb Name="gOdolwaLeftHairRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_HAIR_ROOT" Offset="0xEF64" />
+        <Limb Name="gOdolwaLeftHairBaseLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_HAIR_BASE" Offset="0xEF70" />
+        <Limb Name="gOdolwaLeftLowerHairRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_LOWER_HAIR_ROOT" Offset="0xEF7C" />
+        <Limb Name="gOdolwaLeftHairMiddleLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_HAIR_MIDDLE" Offset="0xEF88" />
+        <Limb Name="gOdolwaLeftHairTipLimb" Type="Standard" EnumName="ODOLWA_LIMB_LEFT_HAIR_TIP" Offset="0xEF94" />
+        <Limb Name="gOdolwaCenterHairRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_CENTER_HAIR_ROOT" Offset="0xEFA0" />
+        <Limb Name="gOdolwaCenterHairBaseLimb" Type="Standard" EnumName="ODOLWA_LIMB_CENTER_HAIR_BASE" Offset="0xEFAC" />
+        <Limb Name="gOdolwaCenterLowerHairRootLimb" Type="Standard" EnumName="ODOLWA_LIMB_CENTER_LOWER_HAIR_ROOT" Offset="0xEFB8" />
+        <Limb Name="gOdolwaCenterHairMiddleLimb" Type="Standard" EnumName="ODOLWA_LIMB_CENTER_HAIR_MIDDLE" Offset="0xEFC4" />
+        <Limb Name="gOdolwaCenterHairTipLimb" Type="Standard" EnumName="ODOLWA_LIMB_CENTER_HAIR_TIP" Offset="0xEFD0" />
+
+        <!-- Odolwa Skeleton -->
+        <Skeleton Name="gOdolwaSkel" Type="Flex" LimbType="Standard" LimbNone="ODOLWA_LIMB_NONE" LimbMax="ODOLWA_LIMB_MAX" EnumName="OdolwaLimb" Offset="0xF0A8" />
+
+        <!-- Odolwa Animations -->
+        <Animation Name="gOdolwaGlitchyFlailAnim" Offset="0xF918" /> <!-- Unused and glitchy. Original name might be "jmp_crym" -->
```

Comment:
I'm not sure about naming the animation "Glitchy", _probably_ just saying it in the comment should be enough